translator/sample files/navjoy_data/Work Zone Sample.csv

```diff
@@ -0,0 +1,253 @@
+srNo,sys_gUid,uniqueId,createdBy,createdDate,lastUpdatedBy,lastUpdatedDate,constructionType,streetNameFrom,streetNameTo,routeName,mileMarkerStart,mileMarkerEnd,travelRestriction,plannedStartDate,plannedEndDate,closureStatus,actualStartDate,actualEndDate,permitDocument,permitID,imageUrl,imageName/Description
```

Comment:
I don't see a reference anywhere to this file, are we using it?

Reply:
removed